app/views/home/index.html.erb

```diff
@@ -0,0 +1,7 @@
+<div class='home_container'>
+    <h1>Welcome to Budgetty</h1>
+    <div>
+        <%= link_to 'Sign up', new_user_registration_path, class: 'sign_up' %>
+        <%= link_to 'Sign in',  new_user_session_path , class: 'sign_in' %>
+    </div>    
+</div>
```

Comment:
- [ ] as one of the requirements is to follow the design provided you should follow colors, font-family, font-size and composition, and space between elements.
- Screenshot of the Requirements 
<img width="451" alt="Screenshot 2023-03-23 at 11 59 57" src="https://user-images.githubusercontent.com/61790587/227199009-9390b89c-7adb-4af5-9518-9a98992826e7.png">

- [ ] you should make sure to make the background the same as the one the design 

- [ ] make the buttons look the same as the ones in the design they should be in the same place 

- [ ] with the logo of the app above the button in order to match the design 
- Yours 
<img width="336" alt="Screenshot 2023-03-23 at 11 59 18" src="https://user-images.githubusercontent.com/61790587/227199438-a5d13e1b-5db1-458e-b9a1-c05c457aead8.png">
- Design 
<img width="244" alt="Screenshot 2023-03-23 at 12 00 50" src="https://user-images.githubusercontent.com/61790587/227199496-58f53811-0d88-4787-bd4c-ae9e805b03c1.png">